src/main/java/io/javalin/Javalin.java

```diff
@@ -427,6 +429,17 @@ public Javalin maxBodySizeForRequestCache(long bodySizeInBytes) {
         return this;
     }
 
+    /**
+     * Configure instance to throw exception if request body size larger than maxRequestCacheBodySize.
+     * The default value is false.
+     * The method must be called before {@link Javalin#start()}.
+     */
+    public Javalin enableRequestTooLongException() {
```

Comment:
Do we need to override this method in EmbeddedJavalin?

Reply:
no, there is no need to override this method